internal/hmyapi/blockchain.go

```diff
@@ -271,7 +271,7 @@ func (s *PublicBlockChainAPI) GetValidatorInformation(ctx context.Context, addre
 		"address":                 validator.Address.String(),
 		"stake":                   hexutil.Uint64(validator.Stake.Uint64()),
 		"name":                    validator.Description.Name,
-		"validatingPublicKey":     validator.ValidatingPubKey.SerializeToHexStr(),
+		"validatingPublicKey":     validator.SlotPubKeys,
```

Comment:
This changes the format of `validatingPublicKey` field of the response (was a string, is now an array of strings).  Do we need to update the docs and dependents?  @mikedoan, are you already using this API?

Reply:
This is just a temporary build-fix. The actual api are not finalized and not used either.